sql/size_utils.sql

```diff
@@ -367,7 +367,7 @@ $BODY$;
 
 -- Convenience function to return approximate row count
 --
--- main_table - table or hypertable to get approximate row count for
+-- hypertable - table or hypertable to get approximate row count for
```

Comment:
i think this works for regular PG tables too. (cc @pmwkaa ). so shouldn't use hypertable here. Functions that do not have hypertable/chunks prefix work for other PG objects as well.

Reply:
oh see that the change is only to the comments. so just need to fix the comment.

Reply:
Yes, but I think the parameter name is changed in the SQL file, so we should use `relation` here.

Reply:
Relation sounds good to me as well